kore/src/Kore/Repl.hs

```diff
@@ -113,24 +113,22 @@ runRepl tools simplifier predicateSimplifier axiomToIdSimplifier axioms' claims'
         -> [Axiom level]
         -> ExecutionGraph
         -> Graph.Node
-        -> Simplifier (ExecutionGraph, Bool)
+        -> Simplifier ExecutionGraph
```

Comment:
What happened to the return value?

Reply:
We ended up not using it in most call sites.It should probably have been a `MaybeT` instead. We'll re-evaluate all uses of the `stepper` in the Repl and figure out if we could refactor it a bit.